README.md

```diff
@@ -1,51 +1,158 @@
-# Introduction
+# aws-nitro-enclaves-k8s-device-plugin
 
-The Nitro Enclaves [Device Plugin](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) gives your pods and containers the ability to access the [Nitro Enclaves device driver](https://docs.kernel.org/virt/ne_overview.html). The device plugin works with both [Amazon EKS](https://aws.amazon.com/eks/) and self-managed Kubernetes nodes.
+The Nitro Enclaves [Device Plugin](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) gives your pods and containers the ability to access
+the [Nitro Enclaves device driver](https://docs.kernel.org/virt/ne_overview.html). The device plugin works with both [Amazon EKS](https://aws.amazon.com/eks/) and self-managed Kubernetes nodes.
 
-[AWS Nitro Enclaves](https://aws.amazon.com/ec2/nitro/nitro-enclaves/) is an [Amazon EC2](https://aws-content-sandbox.aka.amazon.com/ec2/) capability that enables customers to create isolated compute environments to further protect and securely process highly sensitive data within their EC2 instances.
+[AWS Nitro Enclaves](https://aws.amazon.com/ec2/nitro/nitro-enclaves/) is
+an [Amazon EC2](https://aws-content-sandbox.aka.amazon.com/ec2/) capability that enables customers to create isolated compute environments to further protect and securely process highly sensitive data within their EC2 instances.
+
+-------
+## Prerequisites
 
-# Prerequisites
 To utilize this device plugin, you will need:
 
-  - A configured Kubernetes cluster.
-  - At least one enclave-enabled node available in the cluster. An enclave-enabled node is an EC2 instance with the **EnclaveOptions** parameter set to **true**. For more information on creating an enclaving an enclave-enabled node, review the using [Nitro Enclaves with EKS user guide](https://docs.aws.amazon.com/enclaves/latest/user/kubernetes.html).
+- A configured Kubernetes cluster.
+- At least one enclave-enabled node available in the cluster. An enclave-enabled node is an EC2 instance with the *
+  *EnclaveOptions** parameter set to **true**. For more information on creating an enclave-enabled EKS worker node,
```

Comment:
This line break does not make sense and breaks formatting.

In general the changes above here seem to be just line break changes that do not follow a consistent scheme (break after each sentence or after X characters). The previous variant with no line breaks per paragraph was not really readable in the editor, but I am not sure this is an improvement.

Reply:
done